cif_core.dic

```diff
@@ -21,19 +21,19 @@ data_CIF_CORE
     and used within the Crystallographic Information Framework (CIF).
 ;
 
-save_CIF_CORE
+save_CIF_CORE_HEAD
 
-    _definition.id                CIF_CORE
+    _definition.id                CIF_CORE_HEAD
     _definition.scope             Category
     _definition.class             Head
-    _definition.update            2014-06-18
+    _definition.update            2024-05-15
     _description.text
 ;
-    The CIF_CORE group contains the definitions of data items that
-    are common to all domains of crystallographic studies.
+    The HEAD category encompasses groups of categories that are common to all
```

Comment:
I think it makes more sense to say "The CIF_CORE_HEAD category is the top-level category for all categories in the CIF_CORE dictionary."

Reply:
Yes, I accidentally left "HEAD" from the earlier set of revisions. I will also update the overall definition as you suggested.

However, this is probably the only place that explicitly defines CORE data items as items that are `to all domains of crystallographic studies`. Should we maybe then incorporate this part into the `_description.text` of the dictionary block?

Reply:
I agree that there should be considerably better description in the `_description.text` of the dictionary block.